content/graphql/mutations.md

```diff
@@ -41,6 +41,35 @@ export class UpvotePostInput {
 
 > info **Hint** The `@InputType()` decorator takes an options object as an argument, so you can, for example, specify the input type's description. Note that, due to TypeScript's metadata reflection system limitations, you must either use the `@Field` decorator to manually indicate a type, or use a [CLI plugin](/graphql/cli-plugin).
 
+> info **Hint** When a field is nullable you need to type it verbosely in order to prevent any runtime issue:
+>
+> ```ts
+> @Field(() => String, {{ '{' }} nullable: true {{ '}' }}) title?: string | null;
```

Comment:
I'm not sure if this is even necessary. The issue and explanation don't seem to be specific to NestJS itself; instead, they appear to pertain more generally to the TypeScript language. 

Language-specific features and fundamentals are best described in the language’s official documentation (in this case, TypeScript docs).

Reply:
If the new change is OK then please resolve this conversation.